build/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-## 0.12.8-dev
+## 1.0.0-dev
+
+### Breaking Changes
+
+- Changed the return type of `Builder.build` from `Future<dynamic>` to
```

Comment:
I think this is mostly breaking for build system implementations right? Might want to call that out that most Builder authors should't be impacted.